src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -2292,5 +2290,168 @@ End Module]]></Document>)
                 Await state.AssertSelectedCompletionItem("T")
             End Using
         End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TargetTypePreselection1() As Task
```

Comment:
We only need 1 (or 2 max) completion handler tests around preselection.  We just should verify that it's on when we expect it to be on and off if we expect it to be off.  Any other verification should be in the tests htat verify the completoin service or providers.